src/components/tests/LanguageController.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+
+import LanguageController from "../LanguageController";
+
+let mockedLanguage = "";
+let mockedChangeLanguage = jest.fn();
+jest.mock("react-i18next", () => {
+  return {
+    useTranslation: () => {
+      return ({
+        i18n: {
+          language: mockedLanguage,
+          changeLanguage: mockedChangeLanguage,
+        }
+      })
+    }
+  }
+});
+
+beforeEach(() => {
+  jest.clearAllMocks();
+});
+
+describe("<LanguageController />", () => {
+  test("should have correct class name with th language", async () => {
```

Comment:
could you move `beforeEach` section at line 22 under `describe` and rename `test` to `it` (also in SelectRoute.test.tsx). Thanks :)